### ISSUE TITLE
Smart-Picker: 500 Internal server error on `ocs/v2.php/apps/assistant/api/v1/stt/transcribeAudio`

### ISSUE THREAD

**Issue by nwinkelstraeter** [nextcloud/assistant#75]
### Which version of assistant are you using?

1.0.8

### Which version of Nextcloud are you using?

v28.0.3

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Firefox 125.0.1, Chromium 122.0.6261.128

### Describe the Bug

When using the smart picker in NC-Talk or a text file to transcribe audio the error message 

> Failed to schedule transcription:
>
>
> failure
> 500
> Internal Server Error

shows up.
This happens for directly recording audio and when choosing an existing file.

Starting a transcription in the assistant works like expected

The error log shows this entry:
```
Exception OCA\Assistant\Controller\SpeechToTextApiController::transcribeAudio(): Argument #1 ($appId) must be of type string, null given, called in /var/www/html/lib/private/AppFramework/Http/Dispatcher.php on line 230 in file '/var/www/html/custom_apps/assistant/lib/Controller/SpeechToTextApiController.php' line 98
Exception thrown: Exception
```

I guess this has something to do with d308f526ae6759eada251db50368515af8c5e5d4

The form-data of the request only includes `audioData`. `appID` and `identifier` are missing.
![image](https://github.com/nextcloud/assistant/assets/81554102/b3b984f0-1a7c-4121-a280-02a295794f7d)

### Expected Behavior

* Audio transcription works using the smart-picker

### To Reproduce

- Schedule audio transcription with smart-picker (Record Audio or Choose Audi file) -> 500 Internal server error

**Comment by julien-nc** [comment]
Thank you for reporting this. It should be fixed in v1.0.9. Could you confirm?

**Comment by nwinkelstraeter** [comment]
Yes, this works now in v1.0.9
